settings/DevHome.Settings/Views/ExtensionSettingsPage.xaml

```diff
@@ -0,0 +1,29 @@
+<!-- Copyright (c) Microsoft Corporation and Contributors. -->
+<!-- Licensed under the MIT License. -->
+
+<Page
+    x:Class="DevHome.Settings.Views.ExtensionSettingsPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:behaviors="using:DevHome.Common.Behaviors"
+    xmlns:views="using:DevHome.Common.Views"
+    xmlns:i="using:Microsoft.Xaml.Interactivity"
+    xmlns:ic="using:Microsoft.Xaml.Interactions.Core"
+    behaviors:NavigationViewHeaderBehavior.HeaderMode="Never"
+    mc:Ignorable="d">
+
+    <ScrollViewer VerticalScrollBarVisibility="Auto" MaxWidth="{ThemeResource MaxPageContentWidth}" Margin="{ThemeResource ContentPageMargin}">
+        <StackPanel x:Name="ContentArea">
+            <BreadcrumbBar x:Name="BreadcrumbBar" ItemsSource="{x:Bind ViewModel.Breadcrumbs}" ItemClicked="BreadcrumbBar_ItemClicked" Margin="0,0,0,16" />

```

Comment:
We want the headers to be "sticky" on all the pages. I did a change earlier to make the tops of the settings file consistent, you might want to take that format.

```
    <Grid
        MaxWidth="{ThemeResource MaxPageContentWidth}"
        Margin="{ThemeResource ContentPageMargin}">
        <Grid.RowDefinitions>
            <RowDefinition Height="Auto" />
            <RowDefinition Height="*" />
        </Grid.RowDefinitions>

        <BreadcrumbBar
            x:Name="BreadcrumbBar"
            Margin="0,0,0,16"
            ItemClicked="BreadcrumbBar_ItemClicked"
            ItemsSource="{x:Bind Breadcrumbs}" />

        <ScrollViewer Grid.Row="1" VerticalAlignment="Top">
<!-- the rest -->
```

Reply:
Fixed!